### PR TITLE
rustfmt some submodules of libcore

### PR DESCRIPTION
**Pull request by mkpankov**
[Review on Reviewable](https://reviewable.io/reviews/rust-lang/rust/28896)

### CODE REVIEW

File: src/libcore/fmt/num.rs [hunk 1]
```diff
@@ -79,7 +84,9 @@ trait GenericRadix {
                 x = x / base;                  // Deaccumulate the number.
                 *byte = self.digit(n.to_u8()); // Store the digit in the buffer.
                 curr -= 1;
-                if x == zero { break };        // No more digits left to accumulate.
+                if x == zero {
+                    break
+                };        // No more digits left to accumulate.
```
Comment (nrc) [inline src/libcore/fmt/num.rs]: Did you manually fix the above change but not this one? If not then it is probably a weird bug in rustfmt. In any case, could you fixup this comment too please?

**Comment by nrc** [comment]
Thanks for the PR!

**Comment by mkpankov** [comment]
:smiley:

File: src/libcore/fmt/num.rs [hunk 2]
```diff
@@ -70,7 +72,10 @@ trait GenericRadix {
                 x = x / base;                  // Deaccumulate the number.
                 *byte = self.digit(n.to_u8()); // Store the digit in the buffer.
                 curr -= 1;
-                if x == zero { break };        // No more digits left to accumulate.
+                if x == zero {
+                    // No more digits left to accumulate.
+                    break
+                };
```
Comment (alexcrichton) [inline src/libcore/fmt/num.rs]: cc @nrc, perhaps rustfmt could auto-remove these semicolons?

Comment (nrc) [inline src/libcore/fmt/num.rs]: It could and it should! https://github.com/nrc/rustfmt/issues/422

**Pull request merged**